backend/module/eCampApi/test/Rpc/RpcConfigTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace eCampApiTest\Rpc;
+
+use eCamp\LibTest\PHPUnit\AbstractTestCase;
+use Laminas\Config\Factory as ConfigFactory;
+
+/**
+ * @internal
+ */
+class RpcConfigTest extends AbstractTestCase {
+    public function testProducesSameArray() {
+        $before = ConfigFactory::fromFile(__DIR__.'/test.module.config.php');
+
+        $after = ConfigFactory::fromFiles(
+            array_merge(
+                glob(__DIR__.'/../../config/Rpc/*.config.php')
```

Comment:
I don't really like that we now have the config factory for RPC endpoints, but still have to manually update the example array config every time we change something... Can you instead create some fake config for testing that will not change when we change something about the RPC config?

Reply:
Duplikat von https://github.com/ecamp/ecamp3/pull/1027#discussion_r584193494

Reply:
sure